adserver/templates/adserver/reports/all-publishers_uplift.html

```diff
@@ -0,0 +1,15 @@
+{% extends "adserver/reports/all-publishers.html" %}
+{% load humanize %}
+{% load i18n %}
+
+
+{% block title %}{% trans 'Uplift Report' %}{% endblock %}
```

Comment:
Should we add an explainer in here? I know this is just for us but it might be useful.